services/core/CrateHistorian/README.rst

```diff
@@ -0,0 +1,78 @@
+.. _Crate-Historian:
+
```

Comment:
Please change to .. _Crate_Historian like the forwarder does

Reply:
Done